CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 CHANGELOG
 =========
 
+1.14.1dev
+=========
+* bug-fix: rename MXNet argument from accelerators -> accelerator
```

Comment:
* add a newline between ll. 6-7
* also don't you mean `distributions` -> `distribution`?
* does this count as a breaking change?

Reply:
Yes and yes